iceoryx_posh/test/integrationtests/test_popo_chunk_building_blocks.cpp

```diff
@@ -42,7 +42,7 @@ static constexpr uint32_t NUM_CHUNKS_IN_POOL = 3 * iox::MAX_RECEIVER_QUEUE_CAPAC
 static constexpr uint32_t SMALL_CHUNK = 128;
 static constexpr uint32_t CHUNK_META_INFO_SIZE = 256;
 static constexpr size_t MEMORY_SIZE = NUM_CHUNKS_IN_POOL * (SMALL_CHUNK + CHUNK_META_INFO_SIZE);
-alignas(64) uint8_t g_memory[MEMORY_SIZE];
+alignas(64) static uint8_t g_memory[MEMORY_SIZE];
```

Comment:
As an interested reader I am wondering on the origins of the `alignas(64)` (in special this 'magic number' 64).
Somebody shading a light onto the unilluminated, maybe @MatthiasKillat

Reply:
maybe something connected to _SC_PAGESIZE (but hard-coded)?